lib/pages/home/news_feed_card.dart

```diff
@@ -16,17 +16,28 @@ class NewsFeedCard extends StatelessWidget {
       showMoreButtonKey: const ValueKey('show_more_news_feed'),
       onShowMore: () => Navigator.of(context).pushNamed(Routes.newsFeed),
       future: Provider.of<NewsProvider>(context).fetchNewsFeedItems(limit: 2),
-      builder: (newsFeedItems) => Column(
-          children: newsFeedItems
-              .map((item) => ListTile(
-                    title: Text(item.title),
-                    subtitle: Text(item.date),
-                    trailing: const Icon(Icons.arrow_forward_ios),
-                    contentPadding: EdgeInsets.zero,
-                    dense: true,
-                    onTap: () => Utils.launchURL(item.link, context: context),
-                  ))
-              .toList()),
+      builder: (newsFeedItems) {
+        if (newsFeedItems == null || newsFeedItems.isEmpty) {
+          return Center(
+              child: Text(
+                S.of(context).warningNoneYet,
+                style: TextStyle(color: Theme.of(context).disabledColor),
+              ));
+        }
+

```

Comment:
I don't think this is necessary, it's already handled in the InfoCard

Reply:
True, but only when newsFeedItems is null. When it's just an empty list it doesn't show the "None yet" message.

Reply:
Right, I remember running into this with the favourite websites card, and I changed the provider to return null if there is no data. On second thought, that's not a very nice solution. A better way would be to just make InfoCard handle that on its own - inside the `snapshot.hasData` if, add a condition like `if ((snapshot.data is Map || snapshot.data is Iterable) && snapshot.data.isEmpty) return noneYet()`. I'd rather not duplicate the `noneYet` code anywhere because if we decide to change it (add an illustration or something nicer for instance), it'll be troublesome.